azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -606,6 +607,30 @@ private void setupDevPod(final Map<String, String> envVariables,
     }
   }
 
+  /**
+   * This method is used to setup environment variable for RPMs. Based on this environment
+   * variable startup scripts for the flow container can download/install them.
+   *
+   * @param envVariables
+   * @param flowParam
+   */
+  @VisibleForTesting
+  static void setupRPMs(final Map<String, String> envVariables,
+      final Map<String, String> flowParam) {
+    if (flowParam == null || flowParam.isEmpty()) {
```

Comment:
Nit: MapUtils.isEmpty(flowParam) will be concie.

Reply:
Thanks. I have updated it to use MapUtils.isEmpty()